en_US/durability/managing-replication.md

```diff
@@ -8,7 +8,7 @@ During the initial setup of the cluster, several configuration parameters influe
 
 ### Replication Factor
 
-The replication factor, controlled with `durable_storage.messages.replication_factor` configuration parameter, determines the number of replicas each shard should have across the cluster. The default value is `3`.
+The replication factor, controlled with `durable_storage.replication_factor` configuration parameter, determines the number of replicas each shard should have across the cluster. The default value is `3`.
```

Comment:
Q. As of emqx/emqx@f4ea994e7f2735e4d824899e448a039bc1777896, isn't `replication_factor` still per-DB?
https://github.com/emqx/emqx/blob/f4ea994e7f2735e4d824899e448a039bc1777896/apps/emqx/src/emqx_ds_schema.erl#L327